src/interfaces/ecpg/test/meson.build

```diff
@@ -0,0 +1,54 @@
+# TODO: ECPG is not enabled for windows yet
+if host_machine.system() == 'windows'
+  subdir_done()
+endif
```

Comment:
Why is this needed - shouldn't the equivalent bit in src/interfaces/ecpg/meson.build already prevent getting here?